src/components/send-money/SendMoneyAmountInput.js

```diff
@@ -55,58 +60,70 @@ const CustomDiv = styled(`div`)`
    }
 `
 
+const Big = require('big.js')
 class SendMoneyAmountInput extends Component {
    state = {
-      amount: `${this.props.defaultAmount}` || '',
+      amountInput: `${this.props.defaultAmount}` || '',
       amountStatus: ''
    }
 
+   isDecimalString = (value) => {
+      let REG = /^[0-9]*(|[.][0-9]{1,5})$/
+      return REG.test(value)
+   }
+
    handleChangeAmount = (e, { name, value }) => {
-      const amountStatus = !Number.isInteger(Number(value))
-         ? 'Please enter a whole number.'
-         : value > Number(this.props.amount)
-            ? 'Not enough tokens.' 
-            : ''
+      let amountStatus = ''
+      if (value && !this.isDecimalString(value)) {
+         amountStatus = 'NO MORE THAN 5 DECIMAL DIGITS'
```

Comment:
won't this show  'NO MORE THAN 5 DECIMAL DIGITS' when I enter letters instead of digits? Probably should disallow typing (or copy-pasting) anything besides digits and decimal point / commas.

Reply:
You should try letters and other things, it is already forbidden that